requests_aws4auth/__init__.py

```diff
@@ -208,4 +208,4 @@
 del aws4signingkey
 del exceptions
 
-__version__ = '0.9'
+__version__ = '0.8.1'
```

Comment:
Oh yes, I played with versions on my fork, and I remembered it was 0.8.
Will update